posthog/warehouse/api/saved_query.py

```diff
@@ -61,7 +66,17 @@ def create(self, validated_data):
         except Exception as err:
             raise serializers.ValidationError(str(err))
 
-        view.save()
+        with transaction.atomic():
+            view.save()
+
+            try:
+                DataWarehouseModelPath.objects.create_from_saved_query(view)
+            except Exception:
+                # For now, do not fail saved query creation if we cannot model-ize it.
+                # Later, after bugs and errors have been ironed out, we may tie these two
+                # closer together.
+                logger.exception("Failed to create model path when creating view %s", view.name)
```

Comment:
Being safe here, didn't want to disrupt saved query functionality in case of bugs.